lib/controller.js

```diff
@@ -744,7 +744,7 @@ controller.root = function(opts) {
       // Spawn a new SSH process
       var child = cp.spawn('ssh', ['-i',
         // Use the provided key path
-        opts.key,
+        Tessel.LOCAL_AUTH_KEY,
         // Connect to the Tessel's IP Address
```

Comment:
@rwaldron from my point of view this doesn't make sense (also if it works).
@johnnyman727 thanks for that suggestion but could you explain why this is a better solution ?